spartan/aztec-node/templates/_pod-template.yaml

```diff
@@ -0,0 +1,236 @@
+{{- define "chart.podTemplate" }}
+metadata:
+  labels:
+    {{- include "chart.selectorLabels" . | nindent 4 }}
+    app: node
+spec:
+  {{- if .Values.serviceAccount.create }}
+  serviceAccountName: {{ include "chart.serviceAccountName" . }}
+  {{- end }}
+
+  dnsPolicy: ClusterFirstWithHostNet
```

Comment:
shouldn't this depend on if hostNetwork is set?

Reply:
Hm, maybe. I just copied the pod template out of statefulset so I could use reuse in a deployment